### PR TITLE
Add output to create_vm_role workflow

### PR DESCRIPTION
**Pull request by m4dcoder**

### CODE REVIEW

File: actions/workflows/create_vm_role.yaml [hunk 1]
```diff
@@ -28,16 +39,16 @@ st2cd.create_vm_role:
             input:
                 key: ami_<% $.distro %>
             on-success:
-                - check_cname_if_it_already_exists
```
Comment (humblearner) [inline actions/workflows/create_vm_role.yaml:31]: I kept this as `check_cname` but was asked to change to `check_cname_if_it_already_exists` by @lakshmi-kannan.

Comment (m4dcoder) [inline actions/workflows/create_vm_role.yaml:31]: lol.  i would ignore in this case as i'm doing.

Comment (m4dcoder) [inline actions/workflows/create_vm_role.yaml:31]: just to clarify, i feel the task name is over descriptive.  check_cname is pretty descriptive to me already.

Comment (m4dcoder) [inline actions/workflows/create_vm_role.yaml:31]: And if we want more explanation, we can always add comments at the top of the task so there's no need to make the task name unnecessary long...

**Comment by humblearner** [comment]
Also with `task-default` section at the beginning. We should be overriding the `on-error` task-defaults for tasks before `run-instance` right ? As per your comment: https://github.com/StackStorm/st2cd/pull/190#discussion_r71618330. Just trying to get a better picture of how YAQL works.

**Comment by m4dcoder** [comment]
IMHO, it doesn't matter.  st2cd.destroy_vm is idempotent.  This makes the code cleaner.

**Comment by humblearner** [comment]
Indeed thats the case, code is cleaner. But all the tasks will now make extra call to `destroy_vm`on failure, even when it doesn't exist, prolonging the workflow by little. But as you wish to implement. Thanks for clarifying. :)

**Comment by m4dcoder** [comment]
There's only 1 task (wait_for_instance) that's making the extra call and it's only invoking on error in which the percentage is low which is acceptable.

**Comment by m4dcoder** [comment]
>  all the tasks will now make extra call to destroy_vm on failure

@humblearner: I think you're missing some information.

**Pull request merged**